libs/assertion/src/assertion.cpp

```diff
@@ -0,0 +1,48 @@
+//  Copyright (c) 2019 Thomas Heller
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <hpx/assertion.hpp>
+
+#include <iostream>
+#include <string>
+
+namespace hpx { namespace assertion {
+    namespace {
+        assertion_handler& get_handler()
+        {
+            static assertion_handler handler = nullptr;
+            return handler;
+        }
+    }
+
+    void set_assertion_handler(assertion_handler handler)
```

Comment:
Should we export the functions in this file?

Reply:
Already exported in `libs/assertion/include/hpx/assertion.hpp`